sdks/java/io/solr/src/test/java/org/apache/beam/sdk/io/solr/SolrIOTestUtils.java

```diff
@@ -37,10 +37,8 @@ static void createCollection(
       String collection, int numShards, int replicationFactor, AuthorizedSolrClient client)
       throws Exception {
     CollectionAdminRequest.Create create =
-        new CollectionAdminRequest.Create()
-            .setCollectionName(collection)
-            .setNumShards(numShards)
-            .setReplicationFactor(replicationFactor)
+        CollectionAdminRequest.Create.createCollection(
+                collection, null, numShards, replicationFactor)
```

Comment:
Nit: can this use `public static Create createCollection(String collection, int numShards, int numReplicas)` to avoid `null`

https://github.com/apache/lucene-solr/blob/master/solr/solrj/src/java/org/apache/solr/client/solrj/request/CollectionAdminRequest.java#L395

Reply:
Good catch! Done.